easygcm-sample/src/main/java/eu/inloop/easygcm/sample/MainActivity.java

```diff
@@ -13,7 +13,7 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.main);
         EasyGcm.setLoggingLevel(EasyGcm.Logger.LEVEL_WARNING);
-        EasyGcm.init(this);
+        EasyGcm.init(getApplicationContext());
```

Comment:
I am not sure this is necessary. The library does not hold any references to the Context object after the `init(Context context)` method finishes.